template.yaml

```diff
@@ -3,10 +3,14 @@ Transform: AWS::Serverless-2016-10-31
 Description: >
   NVA Search Resources API and Handler
 
+# More info about Globals: https://github.com/awslabs/serverless-application-model/blob/master/docs/globals.rst
 Globals:
   Function:
     Timeout: 60
     MemorySize: 1800
+    Runtime: java17
+    Architectures:
+      - arm64
```

Comment:
why arm64? snapstart if wanting to implement does not support it. Does arm64 required fewer resources? i dont remember why we wanted to use it...

Reply:
up to 40% better price/performance